src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="io.doorbell.android" >
-    
+
     <uses-permission android:name="android.permission.INTERNET" />
-    
+
     <uses-sdk
-        android:minSdkVersion="3"
-        android:targetSdkVersion="18" />
+        android:minSdkVersion="4"
```

Comment:
When I upgraded to `23`, the IDE suggested I moved to `4`. I kept it as low as possible.

In the future, I suggest add apachage manually so you can have full Android 23+ support.